python/src/autogenstudio/web/routes/tool_servers.py

```diff
@@ -12,6 +12,9 @@
 @router.get("/")
 async def list_servers(user_id: str, db=Depends(get_db)) -> Dict:
     response = db.get(ToolServer, filters={"user_id": user_id})
+    if response.status and response.data:
+        # Sort servers by name
+        response.data.sort(key=lambda x: x.name or '')
```

Comment:
This is incorrect and will result in an exception. There is no `name` in the ToolServer data model.